### ISSUE TITLE
$alert-danger-color / $red makes alert-danger panel not contrast compliant.

### ISSUE THREAD

**Issue by Yalgie** [paypal/bootstrap-accessibility-plugin#90]
$alert-danger-color / $red makes alert-danger panel not WCAG AA compliant.

The original bootstrap colour "#a94442" passes the AA test but the $red: #d2322d; in your variables.scss fails the AA test.

-------

Pass using #a94442:
![screen shot 2016-01-07 at 1 35 45 pm](https://cloud.githubusercontent.com/assets/11155692/12161609/62cc9f1a-b543-11e5-8562-53911ab21366.png)

-------

Fail using #d2322d:
![screen shot 2016-01-07 at 1 36 01 pm](https://cloud.githubusercontent.com/assets/11155692/12161610/62cd7732-b543-11e5-8c6a-44832c13a788.png)

**Comment by mpnkhan** [comment]
Somewhere in commits the CSS has changed.  
https://github.com/paypal/bootstrap-accessibility-plugin/commit/bf75f73dae4c40cb90be866a70f39766c627f714

Anyways , please feel free to raise a PR or use original BS.
